services/minespace-web/src/components/Forms/noticeOfDeparture/EditNoticeOfDepartureForm.js.rej

```diff
@@ -0,0 +1,7 @@
+diff a/services/minespace-web/src/components/Forms/noticeOfDeparture/EditNoticeOfDepartureForm.js b/services/minespace-web/src/components/Forms/noticeOfDeparture/EditNoticeOfDepartureForm.js	(rejected hunks)
```

Comment:
what file is this?

Reply:
😕 I have no idea... That's so weird.  I'll take a look